### PR TITLE
kernel_features: fix tests for new KernelConfig

### PR DESCRIPTION
**Pull request by rafaeldtinoco**
This is a preparation for a fix for: aquasecurity/tracee#851

### CODE REVIEW

File: helpers/kernel_features.go [hunk 1]
```diff
@@ -211,22 +214,20 @@ func (k *KernelConfig) initKernelConfig() error {
...
+	configFile, err := os.Open(filePath)
 	if err != nil {
-		return fmt.Errorf("could not open /proc/config.gz: %w", err)
+		return fmt.Errorf("could not open %v: %w", filePath, err)
 	}
 
 	return k.readConfigFromGZScanner(configFile)
 }
 
 // readConfigFromScanner prepares io.Reader for readConfigFromScanner (/boot/config-$(uname -r))
```
[the remark this answers is not in the document]
Comment (grantseltzer) [reply helpers/kernel_features.go:226]: Ah ok, we could make testing this  easier if `initKernelConfig` took the config file paths that are currently hardcoded (/boot/config-_ and /proc/config.gz) as parameters

[the remark this answers is not in the document]
Comment (rafaeldtinoco) [reply helpers/kernel_features.go:226]: Yep, I had worked a bit on that, catching up now.. that is exactly where I was heading.

[the remark this answers is not in the document]
Comment (rafaeldtinoco) [reply helpers/kernel_features.go:226]: I'm fixing the readConfigFromProcConfigGZ (fyio) as it is broken (discovered during tests).

**Comment by rafaeldtinoco** [comment]
All good here using the files from the testdata directory:

```
/usr/lib/go-1.16/bin/go tool test2json -t /tmp/GoLand/___TestGetKernelConfigValue_in_github_com_aquasecurity_libbpfgo_helpers.test -test.v -test.paniconexit0 -test.run ^\QTestGetKernelConfigValue\E$
=== RUN   TestGetKernelConfigValue
=== RUN   TestGetKernelConfigValue/option_ok
=== RUN   TestGetKernelConfigValue/multiple_options_ok
=== RUN   TestGetKernelConfigValue/multiple_options_ok_with_single_not_ok
=== RUN   TestGetKernelConfigValue/multiple_options_ok_with_multiple_not_ok
=== RUN   TestGetKernelConfigValue/undefined_value
--- PASS: TestGetKernelConfigValue (0.00s)
    --- PASS: TestGetKernelConfigValue/option_ok (0.00s)
    --- PASS: TestGetKernelConfigValue/multiple_options_ok (0.00s)
    --- PASS: TestGetKernelConfigValue/multiple_options_ok_with_single_not_ok (0.00s)
    --- PASS: TestGetKernelConfigValue/multiple_options_ok_with_multiple_not_ok (0.00s)
    --- PASS: TestGetKernelConfigValue/undefined_value (0.00s)
PASS
```

**Review by grantseltzer — APPROVED** [review]
LGTM :-)

**Pull request merged**